src/ValueNodeWrapper.tsx

```diff
@@ -403,15 +403,14 @@ const getDataType = (value: unknown, customNodeData?: CustomNodeData) => {
   return 'invalid'
 }
 
-const getInputComponent = (dataType: DataType, inputProps: InputProps) => {
-  const value = inputProps.value
+const getInputComponent = (data:unknown, dataType: DataType, inputProps: InputProps) => {
```

Comment:
```suggestion
const getInputComponent = (data: JsonData, inputProps: InputProps) => {
  const dataType = getDataType(data)
```
Since we're now passing in `data`, I'd move the `getDataType` call down into here so we don't need to pass both `data` and `dataType` into this function.